src/Plugin/Condition/PathAliasExists.php

```diff
@@ -73,11 +74,19 @@ public static function create(ContainerInterface $container, array $configuratio
   }
 
   /**
-   * {@inheritdoc}
+   * Check if a path alias exists.
+   *
+   * @param string $alias
+   *   The alias to see if exists.
+   * @param mixed $language
+   *   The language to use.
+   *
+   * @return bool
+   *   TRUE if the system path does not match the given alias (ie: the alias
+   *   exists).
    */
-  public function evaluate() {
-    $alias = $this->getContextValue('alias');
-    $language = $this->getContext('language')->hasContextValue() ? $this->getContextValue('language')->getId() : NULL;
+  public function doEvaluate($alias, LanguageInterface $language = NULL) {
+    $language = !is_null($language) ? $language->getId() : NULL;
```

Comment:
Just passing on $language creates the same problem mentioned in PR 263. Specifically, getPathByAlias() wants a string value for the language ID, not the language object itself, or NULL.